src/lib/salesforceAuthenticator.js

```diff
@@ -20,8 +21,38 @@ export async function authenticate (config: Config) {
     get: function (endpoint: string) {
       return request.get({ uri: `${j.instance_url}${endpoint}`, headers: { 'Authorization': `Bearer ${j.access_token}` } })
     },
+    post: function (endpoint: string, form: any) {
+      return new Promise((resolve, reject) => {
+        request.post({
+          uri: `${j.instance_url}${endpoint}`,
+          headers: {
+            'Authorization': `Bearer ${j.access_token}`
+          },
+          formData: form
+        }, function (err, httpResponse, body) {
+          if (err != null) {
+            console.error(err)
+            reject(err)
+          } else {
+            resolve(body)
+          }
+        })
+      })
+    },
+    url: j.instance_url,
+    auth: `Bearer ${j.access_token}`,
     getp: async function (endpoint: string) {
       return rp.get({ uri: `${j.instance_url}${endpoint}`, headers: { 'Authorization': `Bearer ${j.access_token}` } })
+    },
+    getFulfilmentFolder: async function () {
```

Comment:
just a personal preference so feel free to ignore this:

this seems to have different levels of abstraction. On one hand it looks like just handles generic posts and gets.
but it seems it also knows how to do this specific request for the fulfilment folder just because we need to use it in more than one place.
I would prefer to either 
- make the post and get private here and just provide methods for everything like we did with getFulfilmentFolder or
-  keep this generic and reusable providing just post and get and have another module, class object or whatever that uses this to provide all the calls we will need to make and encapsulate all the details of how to make the specific request